app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
```

Comment:
fix imports ordering according to PEP8  https://peps.python.org/pep-0008/#imports